app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -200,10 +200,16 @@ class BatchBackfiller extends RequestHandler {
 
   /// Returns priority for back filled targets.
   ///
+  /// Skips scheduling newly created targets whose available entries are
+  /// less than
```

Comment:
Is this comment complete?

Reply:
Ah, good catch! I missed the `BatchPolicy.kBatchSize`. Updated.